### PR TITLE
Enabling multiple regex replacements

### PR DESCRIPTION
**Pull request by jraysajulga**
Hello! Hoping to implement multiple replacement functionality to these tools so that we can use it on usegalaxy.eu...

### CODE REVIEW

File: tools/text_processing/text_processing/replace_text_in_line.xml [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
+        's/$replacement.find_pattern/$replacement.replace_pattern/g'
+      #end for
       '$infile'
       > '$outfile'
 ]]>
 
     </command>
     <inputs>
-	<param format="txt" name="infile" type="data" label="File to process" />
+     <param format="txt" name="infile" type="data" label="File to process" />
```
Comment (bgruening) [inline tools/text_processing/text_processing/replace_text_in_line.xml]: indentation ...

File: tools/text_processing/text_processing/replace_text_in_line.xml [hunk 2]
```diff
@@ -1,25 +1,29 @@
...
+      <repeat name="replacements" title="Replacement">
```
Comment (bgruening) [inline tools/text_processing/text_processing/replace_text_in_line.xml]: `min="1"`

File: tools/text_processing/text_processing/replace_text_in_column.xml [hunk 3]
```diff
@@ -13,41 +13,62 @@
...
-            </sanitizer>
-        </param>
-        <param name="replace_pattern" type="text" label="Replace with" help="Use simple text, or &amp; (ampersand) and \\1 \\2 \\3 to refer to matched text. See examples below." >
-            <sanitizer>
-                <valid initial="string.printable">
-                    <remove value="&apos;"/>
-                </valid>
-            </sanitizer>
-        </param>
+        <repeat name="replacements" title="Replacement">
```
Comment (bgruening) [inline tools/text_processing/text_processing/replace_text_in_column.xml]: `min="1"`

**Comment by jraysajulga** [comment]
@bgruening Tool version has been upgraded. Thanks for the review!

**Comment by bgruening** [comment]
Thanks @jraysajulga!

**Comment by jraysajulga** [comment]
@bgruening Can you update these tools on galaxy.eu as well?

**Comment by bgruening** [comment]
Done, sorry forgot about that.

**Comment by jraysajulga** [comment]
No problem. Thank you!

**Pull request merged**